go/arrow/array/table.go

```diff
@@ -257,6 +258,32 @@ func (tbl *simpleTable) Release() {
 	}
 }
 
+func (tbl *simpleTable) String() string {
+	o := new(strings.Builder)
+	o.WriteString(tbl.Schema().String())
+	o.WriteString("\n")
+
+	for i := 0; i < int(tbl.NumCols()); i++ {
+		col := tbl.Column(i)
+		chunked := col.Data()
+		chunks := chunked.Chunks()
+		o_array := new(strings.Builder)
+		o_array.WriteString(col.Field().Name + ": [")
```

Comment:
why the need for the separate builder? why not just keep writing to `o` the whole time?

Reply:
yeah we could use the `o`. Added just for the readability.

Reply:
I'd prefer to avoid the extra builder because of the cost of creating a new builder for every column. It would be more readable to avoid the intermediate string builder (not to mention potentially large allocations as it ends up having to deep copy the string)

Reply:
You're correct, I updated the code.